Code Style fixes for administrator/components/com_contact

photodude opened the pull request:
Pull Request for Code Style Issues .
### Summary of Changes
- No blank line found after control structure
- Expected "boolean" but found "bool" for function return type
- Line exceeds 150 characters

[Automatically fixed with PHPCS2 fixers](https://github.com/joomla/coding-standards/pull/109)
### Testing Instructions

merge by Code review
### Documentation Changes Required

none

photodude committed:
Code Style fixes for administrator/components/com_contact

- Closing parenthesis of a multi-line IF statement must be on a new line
- No blank line found after control structure
- Expected "boolean" but found "bool" for function return type
- Line exceeds 150 characters

zero-24 commented on administrator/components/com_contact/models/contacts.php:
```diff
@@ -95,7 +95,8 @@ protected function populateState($ordering = 'a.name', $direction = 'asc')
 		$this->setState('filter.published', $this->getUserStateFromRequest($this->context . '.filter.published', 'filter_published', '', 'string'));
-		$this->setState('filter.category_id', $this->getUserStateFromRequest($this->context . '.filter.category_id', 'filter_category_id', '', 'string'));
+		$this->setState('filter.category_id',
+						$this->getUserStateFromRequest($this->context . '.filter.category_id', 'filter_category_id', '', 'string'));
```
Can we have than on all commas a line break? I guess it is better readable than ;)

photodude replied:
I have no objection to that.

photodude replied:
Although, I'm not sure it makes sense in this case since the next  comma is part of another function call being passed to the first function.

andrepereiradasilva replied:
so tabs count as how much characters now?

photodude replied:
tabs should still count as 4 spaces

zero-24 replied:
Ahh i have missed that. It is ok now after the last commit ;) 👍

photodude committed:
Fix the phpcs 1.5 errors

- Closing parenthesis of a multi-line IF statement must be on the same
line
- Closing parenthesis of a multi-line function call must be on a line by
itself

zero-24 commented:
RTC on review thanks :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11988">issues.joomla.org/joomla-cms/11988</a>.</sub>

wilsonge commented on administrator/components/com_contact/models/contacts.php:
```diff
@@ -95,7 +95,9 @@ protected function populateState($ordering = 'a.name', $direction = 'asc')
 		$this->setState('filter.published', $this->getUserStateFromRequest($this->context . '.filter.published', 'filter_published', '', 'string'));
-		$this->setState('filter.category_id', $this->getUserStateFromRequest($this->context . '.filter.category_id', 'filter_category_id', '', 'string'));
+		$this->setState('filter.category_id',
+						$this->getUserStateFromRequest($this->context . '.filter.category_id', 'filter_category_id', '', 'string')
```
Shouldn't this be 1 extra tab instead of 3?

photodude replied:
Probably, that was a manual change to fix the line length and phpstorm aligned it that way (could be an issue in how I have phpstorm configured)
